esmvaltool/cmorizers/obs/utilities.py

```diff
@@ -145,26 +148,30 @@ def read_cmor_config(dataset):
 def save_variable(cube, var, outdir, attrs, **kwargs):
     """Saver function."""
     # CMOR standard
-    cube_time = cube.coord('time')
-    reftime = Unit(cube_time.units.origin, cube_time.units.calendar)
-    dates = reftime.num2date(cube_time.points[[0, -1]])
-    if len(cube_time.points) == 1:
-        year = str(dates[0].year)
-        time_suffix = '-'.join([year + '01', year + '12'])
-    else:
-        date1 = str(dates[0].year) + '%02d' % dates[0].month
-        date2 = str(dates[1].year) + '%02d' % dates[1].month
-        time_suffix = '-'.join([date1, date2])
-
-    file_name = '_'.join([
+    try:
+        cube_time = cube.coord('time')
+        reftime = Unit(cube_time.units.origin, cube_time.units.calendar)
```

Comment:
It would be great if you could keep the amount of code in a try block to a minimum as recommended by PEP8. Or you could simply use something like `if cube.coords('time')` to check if there is a time coordinate